Sources/SwiftDocC/Infrastructure/Topic Graph/AutomaticCuration.swift

```diff
@@ -123,12 +123,9 @@ public struct AutomaticCuration {
         renderContext: RenderContext?,
         renderer: DocumentationContentRenderer
     ) throws -> TaskGroup? {
-        if let automaticSeeAlsoOption = node.options?.automaticSeeAlsoBehavior
-            ?? context.options?.automaticSeeAlsoBehavior
+        if node.options?.automaticSeeAlsoEnabled ?? context.options?.automaticSeeAlsoEnabled == false
```

Comment:
Readability nit:

```suggestion
        if (node.options?.automaticSeeAlsoEnabled ?? context.options?.automaticSeeAlsoEnabled) == false
```

Reply:
Fixed in https://github.com/apple/swift-docc/pull/448/commits/9925622d9074d84f4f3a93744da4d66dab37c9cb